scripts/docker/mxe-build-container/settings-stage2.mk

```diff
@@ -9,7 +9,31 @@ JOBS := 2
 MXE_TARGETS :=  i686-w64-mingw32.shared
 
 # The three lines below makes `make` build these "local packages" instead of all packages.
-LOCAL_PKG_LIST := qtbase qtconnectivity qtdeclarative qtimageformats qtlocation qtmultimedia qtquickcontrols qtquickcontrols2 qtscript qtsvg qttools qttranslations qtwebview qtwebkit libxml2 libxslt libusb1 libgit2 nsis curl libzip libftdi1
+LOCAL_PKG_LIST := curl \
+									grantlee \
```

Comment:
whitespace looks a little odd :-)

Reply:
Hmm it does look odd in the review. My intention was to put them on separate lines (and sorted) so it was easy to see what packages are in there.
I will clean that up.